java/clients/src/main/java/sleeper/clients/status/report/filestatus/StandardFileStatusReporter.java

```diff
@@ -36,49 +39,83 @@ public StandardFileStatusReporter(PrintStream out) {
 
         if (verbose) {
-            out.print("Files with no references:\n");
-            out.println(fileStatusReport.getFilesWithNoReferences());
-            out.println("Active files:");
-            fileStatusReport.getActiveFiles().forEach(out::println);
+            out.println("Files with no references"
+                    + (status.isMoreThanMax() ? " (more are present, maximum count applied)" : "") + ":"
+                    + (status.getFilesWithNoReferences().isEmpty() ? " none" : ""));
+            status.getFilesWithNoReferences().forEach(this::printFile);
+            out.println("Files with references:"
+                    + (status.getFilesWithReferences().isEmpty() ? " none" : ""));
+            status.getFilesWithReferences().forEach(this::printFile);
         }
         String percentageSuffix = "= ";
         String allActiveFilesSuffix = "= ";
-        if (fileStatusReport.getTotalRecordsApprox() > 0L) {
+        if (status.getTotalRecordsApprox() > 0L) {
             allActiveFilesSuffix = "(approx) = ";
             percentageSuffix = "(approx) = ";
         }
         String leafFilesSuffix = "= ";
-        if (fileStatusReport.getTotalRecordsInLeafPartitionsApprox() > 0L) {
+        if (status.getTotalRecordsInLeafPartitionsApprox() > 0L) {
             leafFilesSuffix = "(approx) = ";
             percentageSuffix = "(approx) = ";
         }
+        String nonLeafFilesSuffix = "= ";
+        if (status.getTotalRecordsInNonLeafPartitionsApprox() > 0L) {
+            nonLeafFilesSuffix = "(approx) = ";
+        }
         out.println("Total number of records in all active files " + allActiveFilesSuffix +
-                abbreviatedRecordCount(fileStatusReport.getTotalRecords()));
+                abbreviatedRecordCount(status.getTotalRecords()));
+        out.println("Total number of records in non-leaf partitions " + nonLeafFilesSuffix +
+                abbreviatedRecordCount(status.getTotalRecordsInNonLeafPartitions()));
         out.println("Total number of records in leaf partitions " + leafFilesSuffix +
-                abbreviatedRecordCount(fileStatusReport.getTotalRecordsInLeafPartitions()));
+                abbreviatedRecordCount(status.getTotalRecordsInLeafPartitions()));
         out.println("Percentage of records in leaf partitions " + percentageSuffix +
-                (fileStatusReport.getTotalRecordsInLeafPartitions() / (double) fileStatusReport.getTotalRecords()) * 100.0);
+                (status.getTotalRecordsInLeafPartitions() / (double) status.getTotalRecords()) * 100.0);
     }
 
-    private void printPartitionStats(FileStatus.PartitionStats partitions, String type) {
-        if (partitions.getTotal() > 0) {
+    private void printPartitionStats(FileReferencesStats partitions, String type) {
+        if (partitions.getTotalReferences() > 0) {
             out.println("Number of files in " + type + " partitions:" +
```

Comment:
This should probably read `Number of file references in leaf/non-leaf partitions`